### DISCUSSION TITLE
The process on github education

### DISCUSSION THREAD

**Discussion by NhatErik76** [community/community#121385] [GitHub Education]
### Select Topic Area

Question

### Body

![image](https://github.com/community/community/assets/160149794/a3db4ac0-c568-4fc4-adbe-1c4aaaf482f8)
how much people have faced the same issue like this. I have to wait for a week and ...
how i can see the process and solve this problem

**Comment by 8xu** [comment]
It may take a bit longer to get accepted, I think you can reach out to support if you have questions or concerns

**Reply by NhatErik76** [reply]
I don't know what link will help me solve this problem can you help me reach out that support link

**Comment by itzzjb** [comment]
It's normal. I had that issue too. But if you can go to a session done by a GitHub campus expert they can give you a QR link to get the student pack within a day.

**Reply by NhatErik76** [reply]
how can i ask them to give me a qr link. Can you give me a link to those experts

**Reply by itzzjb** [reply]
Ahh. For that, you need to attend sessions done by a GitHub campus expert.

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @NhatErik76 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Reply by NhatErik76** [reply]
Hi @queenofcorgis 

Sorry for bothering you. But i have seen  this message "We're still processing your application. Once this is complete you will have access to the student developer pack offers." for more than a week How you can help me speed up the process so that i can access to the education benefits

**Comment by apasyunaru** [comment]
They notify you via e-mail when the process is completed.

**Comment by samus-aran** [comment]
Hi @NhatErik76 

Are you still experiencing this issue? Usually we cancel draft application after 3 days so perhaps you can see if you can re-apply.

**Comment by NhatErik76** [comment]
I'm sorry for not explaining soon but I still have the same problem. "We're
still processing your application. Once this is complete you will have
access to the student developer pack offers."
